### ISSUE TITLE
Support accent in localSort() for grid

### ISSUE THREAD

**Issue by antoninmeunier** [vitmalina/w2ui#1991]
**Describe the problem your feature will solve.**
Sorting the grid with the sort() or localSort() method doesn't handle the accents, for exemple in French : É is sorted after Z, which is not true. 

**Describe the solution you'd like**
Sorting methods should consider accented characters and non-accented characters as the same. 

**Describe alternatives you've considered**
I bypassed this method and created my own method directly in JS file but I guess it should be included in the source code.

**Comment by vitmalina** [comment]
submit a pull request

**Comment by mpf82** [comment]
I think that could easily be done by passing ``Intl.Collator().compare`` as sortMode. Docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/compare

It's supported by all modern browsers: https://caniuse.com/?search=Intl.Collator.compare

We could also add it as an internal feature, as addition to "natural", e.g. by passing ``international`` as sortMode.

Comparison:

```
['1', '10', '2', '20', '3', 'a', 'ä', 'e', 'é', 'i', 'o', 'ö', 'u', 'ú', 'ü',].sort()
['1', '10', '2', '20', '3', 'a', 'e', 'i', 'o', 'u', 'ä', 'é', 'ö', 'ú', 'ü']

['1', '10', '2', '20', '3', 'a', 'ä', 'e', 'é', 'i', 'o', 'ö', 'u', 'ú', 'ü',].sort(w2utils.naturalCompare)
['1', '2', '3', '10', '20', 'a', 'e', 'i', 'o', 'u', 'ä', 'é', 'ö', 'ú', 'ü']

['1', '10', '2', '20', '3', 'a', 'ä', 'e', 'é', 'i', 'o', 'ö', 'u', 'ú', 'ü',].sort(Intl.Collator().compare)
['1', '10', '2', '20', '3', 'a', 'ä', 'e', 'é', 'i', 'o', 'ö', 'u', 'ú', 'ü']
```

However, combining natural sort **and** international sort would be a whole different story.

**Comment by mpf82** [comment]
Closing issue: implemented in PR #2111
